spec/models/task_spec.rb

```diff
@@ -1,5 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+	it "タイトルとステータスがあれば有効なこと" do
+		user = create(:user)
+		task = build(:task, user_id: user.id)
+		expect(task).to be_valid
+	end
+	it "タイトルがなければ無効なこと" do
+		user = create(:user)
+		task = build(:task, user_id: user.id, title: "")
+		task.valid?
+		expect(task.errors[:title]).to include("can't be blank")
+	end
+	it "ステータスがなければ無効なこと" do
+		user = create(:user)
+		task = build(:task, user_id: user.id, status: "")
+		task.valid?
+		expect(task.errors[:status]).to include("can't be blank")
+	end
+	it "タイトルが重複していたら無効なこと" do
+		user = create(:user)
+		task = create(:task, user_id: user.id)
+		task1 = build(:task, user_id: user.id)
+		task1.valid?
```

Comment:
task1という変数名だと、どのようなデータが入っている変数なのかが分からないので、できるだけ意味合いが伝わるような適切な命名をするように心がけましょう。
```suggestion
		another_task = build(:task, user_id: user.id)
		another_task.valid?
```

Reply:
864bac1 でtask1からanother_taskに変数名を修正しました